apps/bc-wallet-showcase-creator/lib/steps.ts

```diff
@@ -1,58 +1,85 @@
-import type { ScenarioRequestType, StepRequestType } from '@/openapi-types'
+import { ScenarioRequestType, StepRequestType } from "@/openapi-types";
 
-export const sampleAction = {
-  title: 'example_title',
-  actionType: 'ARIES_OOB',
-  text: 'example_text',
+export interface StepWithCredentials extends StepRequestType {
+  credentials?: string[];
+}
+
+export const sampleAction =  {
+  title: "example_title",
+  actionType: "ARIES_OOB",
+  text: "example_text",
   proofRequest: {
     attributes: {
       attribute1: {
-        attributes: ['attribute1', 'attribute2'],
-        restrictions: ['restriction1', 'restriction2'],
+        attributes: ["attribute1", "attribute2"],
+        restrictions: ["restriction1", "restriction2"],
       },
     },
     predicates: {
       predicate1: {
-        name: 'example_name',
-        type: 'example_type',
-        value: 'example_value',
-        restrictions: ['restriction1', 'restriction2'],
+        name: "example_name",
+        type: "example_type",
+        value: "example_value",
+        restrictions: ["restriction1", "restriction2"],
       },
     },
   },
 }
 
 export const sampleScenario: ScenarioRequestType = {
-  name: 'example_name',
-  description: 'example_description',
-  type: 'ISSUANCE' as 'ISSUANCE' | 'PRESENTATION',
-  issuer: '3de59a17-222e-4c92-a22a-118eff7032b5',
+  name: "example_name",
+  description: "example_description",
+  type: "ISSUANCE" as "ISSUANCE" | "PRESENTATION",
+  issuer: "3de59a17-222e-4c92-a22a-118eff7032b5",
```

Comment:
this issuer can crash the application if is not on the database, we have a "workaround" when a credential is created